src/stories/colors.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import type { Meta } from '@storybook/react'
+
+import colors from 'styles/sfds/colors.module.scss'
+
+export default {
+  title: 'Global/Colors',
+} as Meta
+
+const filterGroup = (filter: string, omit?: string) =>
+  Object.keys(colors).filter(
+    (color) =>
+      color.indexOf(filter) === 0 && (omit ? color.indexOf(omit) === -1 : color)
+  )
+
+const Color = ({ color }: { color: string }) => (
+  <li
+    style={{
+      borderRadius: '5px',
+      border: '1px solid lightgray',
+      padding: '8px',
+      marginBottom: '12px',
+    }}>
+    <span
+      style={{
+        backgroundColor: colors[color],
+        display: 'block',
+        height: '4em',
+        marginBottom: '0.3em',
+        borderRadius: '5px',
+      }}
+    />
+    <span style={{ fontFamily: 'monospace, monospace' }}>
+      ${color}
+      <br />
+      {colors[color]}
```

Comment:
Same as above; do we want to make this: `{colors[parseInt(color)]}` to clear up this warning?